Improve backup documentation (#4914)
kyma-project/kyma: docs/backup/03-03-restore-backup.md

type: Details
---

> Restoring a Kyma cluster requires installing Velero. Download Velero CLI and use it to install the Velero server. Then, using the client, instruct Velero to start the restore process. Restore the CRDs, Services, and Endpoints first, and then the rest of the resources.

Reviewer: ```suggestion
Restoring a Kyma cluster requires installing Velero. Download Velero CLI and use it to install the Velero server. Then, using the client, instruct Velero to start the restore process. Restore the CRDs, services, and endpoints first, and then the rest of the resources. Follow these steps to restore the resources: 
```

Reviewer: I added the steps because it makes sense (as it is a procedure).

> Download and install [Velero CLI v1.0.0](https://github.com/heptio/velero/releases/tag/v1.0.0).

Reviewer: ```suggestion
1. Download and install [Velero CLI v1.0.0](https://github.com/heptio/velero/releases/tag/v1.0.0).
```

> Install Velero server using the same bucket information where the backups reside:

Reviewer: ```suggestion
2. Install the Velero server. Use the same bucket as for the backups:
```

```
velero install --bucket <BUCKET> --provider <CLOUD_PROVIDER> --secret-file <CREDENTIALS_FILE> --restore-only --wait
```

> List available backups:

Reviewer: ```suggestion
3. List available backups:
```

```
velero get backups
```

> Restore Kyma CRDs, Services, and Endpoints:

Reviewer: ```suggestion
3. Restore Kyma CRDs, services, and endpoints:
```

```
velero restore create --from-backup <BACKUP_NAME> --include-resources customresourcedefinitions.apiextensions.k8s.io,services,endpoints --include-cluster-resources --wait
```

> Restore the rest of Kyma:

Reviewer: ```suggestion
4. Restore the rest of Kyma resources:
```

```
velero restore create --from-backup <BACKUP_NAME> --exclude-resources customresourcedefinitions.apiextensions.k8s.io,services,endpoints --include-cluster-resources --restore-volumes --wait
```

> Once the status of the restore is `COMPLETED`, verify the health of Kyma by checking the Pods:

Reviewer: I would move the code snipped before the note, and remove the note formatting.

> > **NOTE:** Even if the restore process is complete, it may take some time for the resources to become available again.

Reviewer: ```suggestion
Even if the restore process is complete, it may take some time for the resources to become available again.
```
